README.md

```diff
@@ -1 +1,148 @@
-# javascript-capstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Todo List Project](#-Javascript-project)
+  - [🛠 Built With ](#-built-with-)
+  - [🚀 Live Demo video ](https://youtu.be/sgqM34DgA4Q)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Javascript Capstone<a name="about-project"></a>
+
+**javascript Capstone** 
+The JavaScript capstone project  is about building your own web application based on an external API. We select an API (The pokemon api) that provides data about our selected api and we build  the webapp around it. The webapp have 3 user interfaces.
+
+
+## 🛠 Built With <a name="built-with"></a>
+    HTML
+    CSS
+    Javascript(ES6)
+    Webpack
+    AAA patern
+    jest Tests
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+ - [Live Demo video](https://youtu.be/sgqM34DgA4Q)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
```

Comment:
- [ ] Please make sure the home page is responsive, you can see that the user can't use your app on his mobile, you should fix this, you should consider using flex and flex-wrap, if you want to use a grid you should make sure that the images and the font sizes are responsive and looks great on any screen.
![image](https://user-images.githubusercontent.com/74744787/211159833-7cdd5e94-f38f-4d1b-a9a0-ecabc03b4f47.png)

- [ ] Please make use of semantic tags, I think it would be better here if you used an unordered list tag `ul` instead of just using a `div` tag everywhere 
![image](https://user-images.githubusercontent.com/74744787/211159976-548537c1-180e-42c6-b176-308318ba8381.png)

- [ Optional] Please make sure to create a validation for the comment section so the user can't write so long comments or names. 
![image](https://user-images.githubusercontent.com/74744787/211160476-f88615b1-4ea5-4081-8dfd-3a175f1e0587.png)

Reply:
> So, in the wireframe, they do not mention building for mobile

@Hassaanjbaig-code it would be better to consider responsiveness in all your projects, the project being responsive is part of following the design